metals/src/main/scala/scala/meta/internal/metals/debug/MetalsDebugAdapters.scala

```diff
@@ -20,15 +23,19 @@ final class MetalsDebugAdapters {
 
     Option(clientConfig.getPathFormat) match {
       case Some(PathFormat.PATH) =>
-        pathAdapter = path => Paths.get(path).toUri().toString()
+        clientToServerPathAdapter = path => Paths.get(path).toUri().toString()
+        serverToClientPathAdapter = _.toString
       case Some(PathFormat.URI) =>
-        pathAdapter = identity // metals expects an URI
+        clientToServerPathAdapter = identity // metals expects an URI
+        serverToClientPathAdapter = _.toURI.toString
       case _ =>
       // ignore
     }
   }
 
   def adaptLine(line: Long): Long = lineAdapter(line)
 
-  def adaptPath(path: String): String = pathAdapter(path)
+  def adaptPath(path: String): String = clientToServerPathAdapter(path)
+
+  def adataPathForClient(path: AbsolutePath) = serverToClientPathAdapter(path)
```

Comment:
Also maybe it should be `adaptPathForClient`? And maybe `adaptPath` -> `AdaptPathForServer`

Reply:
Done.